homeassistant/components/plugwise/coordinator.py

```diff
@@ -51,6 +82,7 @@ def __init__(self, hass: HomeAssistant, entry: ConfigEntry) -> None:
             timeout=30,
             websession=async_get_clientsession(hass, verify_ssl=False),
         )
+        self.current_entities: set[tuple[str, str]] = {("dummy", "dummy_id")}
```

Comment:
- I don't think dummy was supposed to be here?
- Also, it shouldn't be needed to track these? Feels weird?

Reply:
On the use of "dummy", the init needs to have the strings in place, I could change it to `{("","")} that works too.
But `{()}' doesn't work.

Not needed to track these: you mean track the `self.current_entities`? 
This self contains the set of actual entities created during the initialization of the integration.
This set is compared to what is present in the entity_registry, if there are more entities present in the entity_registry than in the set, it means some cleaning can be done.

Reply:
@frenck I'm seeing that ISY994 uses this as init:
```
        current_unique_ids: set[tuple[Platform, str]] = {
            (Platform.BUTTON, f"{self.uuid}_query")
        }
```
I can follow this idea and create a better but still a dummy entry in the set.
The dummy can be there, it does not harm anything because the difference-function in this `set(entities.keys()).difference(isy_data.unique_ids)` only leaves the entities that are present in the entity_registry and are not present in the set with current_unique_ids.

Reply:
Some improvements have been implemented.

Reply:
@frenck looking forward to your feedback on the latest changes :)